portfolio/src/main/webapp/script.js

```diff
@@ -31,3 +32,19 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+function makeCollapsible() {
+    var coll = document.getElementsByClassName("collapsible");
+      var i;
```

Comment:
If you're not referencing `i` outside of the loop, I think you can do:
`for (var i = 0; i < coll.length; i++)`